src/azure-cli/azure/cli/__main__.py

```diff
@@ -81,7 +81,7 @@ def cli_main(cli, args):
                 if datetime.datetime.now() > version_update_time + datetime.timedelta(days=11):
                     get_cached_latest_versions()
                 from packaging.version import parse
-                if parse(VERSIONS['versions']['core']['local']) < parse(VERSIONS['versions']['core']['pypi']):  # pylint: disable=line-too-long
+                if 'pypi' in VERSIONS['versions']['core'] and parse(VERSIONS['versions']['core']['local']) < parse(VERSIONS['versions']['core']['pypi']):  # pylint: disable=line-too-long
```

Comment:
I think we should fix the invalid `~/.azure/versionCheck.json`, instead of here.

Reply:
This file is refreshed after 11 days, so I'd like to leave it there.

Skipping it or setting a dummy value for `pypi` produces the same result. I don't want to make the logic more complicate.